tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -4149,8 +4149,13 @@ Status ConvertBinary(OpConverterParams* params) {
                                    " inputs but expected 2, at ",
                                    node_def.name());
   }
+#if IS_TRT_VERSION_GE(6, 0, 0, 0)
```

Comment:
A better way is to use a static array to hold the data types and only this array value is conditional on TRT version.